opt/scopeserver/dataserver/utils/search.py

```diff
@@ -54,26 +54,39 @@ class ResultTypePair(NamedTuple):
 }
 
 
-def match_result_cost(term: str, result: str) -> Optional[int]:
+def match_result_cost(term: str, result: str, result_type: str) -> Tuple[int, Optional[int]]:
```

Comment:
This makes using the result more difficult. Could you sum these values? Would that be useful? Then you can just return an int.